pkgs/top-level/all-packages.nix

```diff
@@ -19221,7 +19221,25 @@ with pkgs;
 
   norminette = callPackage ../development/tools/norminette { };
 
-  nwjs = callPackage ../development/tools/nwjs { };
+  nwjs = callPackage ../development/tools/nwjs {
+    version = "0.85.0";
```

Comment:
Is there a reason nwjs is reverted to 0.85.0? Only betaflight-configurator has issues prompting the need for 0.84.0 correct?

Reply:
Also all-packages may not be the best place for this, perhaps nwjs can be reworked to allow overrideAttrs?

Reply:
rolling back to 0.85 was just a mistake, I've fixed it now

Reply:
A number of packages were broken by an upgrade to 0.85:
```
blheli-configurator (new package, https://github.com/NixOS/nixpkgs/pull/310820)
betaflight-configurator
inav-blackbox-explorer (new package, https://github.com/NixOS/nixpkgs/pull/310813)
```
Probably `betaflight-blackbox-explorer` is affected, too (I haven't packaged it yet)

Reply:
regarding overrideAttrs, I don't see how it's possible to make it work without code duplication. We're interpolating the version in the url, so we'd have to do it again with a new value for version.
I made version and hashes the same kind of input as `sdk`. Also, python uses a similar approach to allow for multiple versions